docs/source/external_tutorials/tutorial_cellpose_segmentation.ipynb

```diff
@@ -0,0 +1,400 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Nuclei segmentation using cellpose"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial shows how we can easily use an external segmentation packages in the *Squidpy* framework."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import squidpy as sq\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare cellpose functionality for *Squidpy*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use the cellpose package, we need to install it with `pip install cellpose`.\n",
+    "For details on the segmentation model, see: Stringer, Carsen, et al. \"Cellpose: a generalist algorithm for cellular segmentation.\" Nature Methods (2021).\n",
+    "Or check out https://github.com/MouseLand/cellpose for the python implementation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the cellpose segmentation model. See https://cellpose.readthedocs.io/en/latest/api.html#cellpose-class.\n",
+    "\n",
+    "from cellpose import models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The method parameter of the `sq.im.segment` method accepts any callable with the signature:\n",
+    "`numpy.ndarray` (height, width, channels) -> `numpy.ndarray` (height, width[, channels]).\n",
+    "Additional model specific arguments will also be passed on.\n",
+    "To use the cellpose model, we define a wrapper that initializes the model, evaluates it and returns the\n",
+    "segmentation mask. We can make use of cellpose specific options by passing on arguments like the minimum\n",
+    "number of pixels per mask `min_size`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Wrap the cellpose model initialization and evaluation into one method that we can then pass on to the segmentation\n",
```

Comment:
I'd suggest to put these comments (and others like them below) in markdown cells. This renders much prettier I think.